test_requirements.txt

```diff
@@ -1,7 +1,2 @@
-pytest==5.4.3; python_version>="3.6"  # Fixed in https://github.com/pytest-dev/pytest/pull/7565
-pytest; python_version<="2.7"
-mock
-zipp<2; python_version<='2.7'
-configparser<5; python_version<='2.7'
```

Comment:
Why where these in place at all? `mock` isn't used? Configparser only for py2?

Reply:
To be honest, no, I dont know. I suspect they are remnants from the creation of this package.